api/onnx_web/chain/source_txt2img.py

```diff
@@ -45,7 +46,17 @@ def run(
             params
         )
 
-        latents = get_latents_from_seed(params.seed, size, params.batch)
+        tile_size = params.tiles
+        if max(size) > tile_size:
```

Comment:
We were talking about ways to tile the latents in Discord, and I'm worried that generating the latents from the seed for each tile (rather than once and tiling those) will produce many copies of the same small image. I will take a look at that

Reply:
Yes, that's the behavior that I've observed. Changing to tiled latents will probably improve things. I mostly left it this way because it felt like the most I could personally do.

I would recommend accepting this PR and #395 before working on this, because both of these make substantial changes to tile.py works. Under this PR, "old" txt2img still works exactly the same if tile size > image size